### PR TITLE
refactor(file transfer): adapt new fields in ssl config

### PR DESCRIPTION
**Pull request by Kinplemelon**

### CODE REVIEW

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-2618)

File: src/schemaText/schema-text-zh.json [hunk 1]
```diff
@@ -1373,6 +1373,14 @@
     "log_level": {
       "desc": "SSL 通信的日志级别。默认为 '通知'。设置为 'debug' 可检查 TLS 握手信息",
       "label": "SSL 日志级别"
+    },
+    "partial_chain": {
+      "desc": "启用或禁用使用 partial_chain 进行对等验证。当本地在 x509 路径验证过程中验证对等证书时，它会构建一个从对等证书开始，以信任锚结束的证书链。默认情况下，如果设置为 `false`，信任锚是根 CA，证书链必须完整。然而，如果设置为 `true` 或 `cacert_from_cacertfile`，则 `cacertfile` 中的最后一个证书将被用作信任锚证书（中间 CA）。这在路径验证中创建了一个部分链。或者，如果配置为 `two_cacerts_from_cacertfile`，`cacertfile` 中的最后两个证书之一将被用作信任锚证书，形成部分链。此选项对于中间 CA 证书轮换特别有用。然而，请注意，它会带来一些额外的开销，因此应仅用于证书轮换目的。",
+      "label": "部分链"
```
Comment (wivwiv) [inline src/schemaText/schema-text-zh.json:1379]:
```suggestion
"label": "部分链验证"
```

Comment (wivwiv) [inline src/schemaText/schema-text-zh.json:1379]:
```suggestion
      "label": "启用部分链验证"
```

File: src/schemaText/schema-text-zh.json [hunk 2]
```diff
@@ -1373,6 +1373,14 @@
...
+    },
+    "verify_peer_ext_key_usage": {
+      "desc": "验证对等证书中的扩展密钥用法<br/>为了进行额外的对等证书验证，这里定义的值必须出现在 [rfc5280](https://www.rfc-editor.org/rfc/rfc5280#section-4.2.1.12) 中定义的对等证书的“扩展密钥用法”中。<br/>允许的值有<br/>- `clientAuth`<br/>- `serverAuth`<br/>- `codeSigning`<br/>- `emailProtection`<br/>- `timeStamping`<br/>- `ocspSigning`<br/>- 原始 OID，例如：\"OID:1.3.6.1.5.5.7.3.2\" 意味着 `id-pk 2`，相当于 `clientAuth`<br/>也支持逗号分隔的字符串来验证多个密钥用法。<br/>例如，`\"serverAuth,OID:1.3.6.1.5.5.7.3.2\"`",
+      "label": "验证证书中的密钥用法"
```
Comment (wivwiv) [inline src/schemaText/schema-text-zh.json:1383]:
```suggestion
      "label": "验证扩展密钥用法"
```

File: src/schemaText/schema-text-zh.json [hunk 3]
```diff
@@ -1373,6 +1373,14 @@
     "log_level": {
       "desc": "SSL 通信的日志级别。默认为 '通知'。设置为 'debug' 可检查 TLS 握手信息",
       "label": "SSL 日志级别"
+    },
+    "partial_chain": {
+      "desc": "启用或禁用使用 partial_chain 进行对等验证。当本地在 x509 路径验证过程中验证对等证书时，它会构建一个从对等证书开始，以信任锚结束的证书链。默认情况下，如果设置为 `false`，信任锚是根 CA，证书链必须完整。然而，如果设置为 `true` 或 `cacert_from_cacertfile`，则 `cacertfile` 中的最后一个证书将被用作信任锚证书（中间 CA）。这在路径验证中创建了一个部分链。或者，如果配置为 `two_cacerts_from_cacertfile`，`cacertfile` 中的最后两个证书之一将被用作信任锚证书，形成部分链。此选项对于中间 CA 证书轮换特别有用。然而，请注意，它会带来一些额外的开销，因此应仅用于证书轮换目的。",
+      "label": "部分链"
+    },
+    "verify_peer_ext_key_usage": {
+      "desc": "验证对等证书中的扩展密钥用法<br/>为了进行额外的对等证书验证，这里定义的值必须出现在 [rfc5280](https://www.rfc-editor.org/rfc/rfc5280#section-4.2.1.12) 中定义的对等证书的“扩展密钥用法”中。<br/>允许的值有<br/>- `clientAuth`<br/>- `serverAuth`<br/>- `codeSigning`<br/>- `emailProtection`<br/>- `timeStamping`<br/>- `ocspSigning`<br/>- 原始 OID，例如：\"OID:1.3.6.1.5.5.7.3.2\" 意味着 `id-pk 2`，相当于 `clientAuth`<br/>也支持逗号分隔的字符串来验证多个密钥用法。<br/>例如，`\"serverAuth,OID:1.3.6.1.5.5.7.3.2\"`",
```
Comment (wivwiv) [inline src/schemaText/schema-text-zh.json:1382]:
```suggestion
      "desc": "验证对等证书中的扩展密钥用法<br/>为了进行额外的对等证书验证，这里定义的值必须使用 [rfc5280](https://www.rfc-editor.org/rfc/rfc5280#section-4.2.1.12) 中定义的对等证书的“扩展密钥用法”。<br/>允许的值有<br/>- `clientAuth`<br/>- `serverAuth`<br/>- `codeSigning`<br/>- `emailProtection`<br/>- `timeStamping`<br/>- `ocspSigning`<br/>- 原始 OID，例如：\"OID:1.3.6.1.5.5.7.3.2\" 意味着 `id-pk 2`，相当于 `clientAuth`<br/>也支持逗号分隔的字符串来验证多个密钥用法。<br/>例如，`\"serverAuth,OID:1.3.6.1.5.5.7.3.2\"`",
```

File: src/schemaText/schema-text-en.json [hunk 4]
```diff
@@ -1381,6 +1381,14 @@
...
       "label": "Cipher Suites"
+    },
+    "partial_chain": {
+      "desc": "Enable or disable peer verification with partial_chain.When local verifies a peer certificate during the x509 path validation process, it constructs a certificate chain that starts with the peer certificate and ends with a trust anchor.By default, if it is set to `false`, the trust anchor is the Root CA, and the certificate chain must be complete.However, if the setting is set to `true` or `cacert_from_cacertfile`, the last certificate in `cacertfile` will be used as the trust anchor certificate (intermediate CA). This creates a partial chain in the path validation.Alternatively, if it is configured with `two_cacerts_from_cacertfile`, one of the last two certificates in `cacertfile` will be used as the trust anchor certificate, forming a partial chain. This option is particularly useful for intermediate CA certificate rotation.However, please note that it incurs some additional overhead, so it should only be used for certificate rotation purposes.",
+      "label": "Partial Chain"
```
Comment (wivwiv) [inline src/schemaText/schema-text-en.json:1387]:
```suggestion
      "label": "Enable Partial Chain Verification"
```

File: src/schemaText/schema-text-en.json [hunk 5]
```diff
@@ -1381,6 +1381,14 @@
...
+    },
+    "verify_peer_ext_key_usage": {
+      "desc": "Verify extended key usage in peer's certificate<br/>For additional peer certificate validation, the value defined here must present in the 'Extended Key Usage' of peer certificate defined in [rfc5280](https://www.rfc-editor.org/rfc/rfc5280#section-4.2.1.12).<br/>Allowed values are<br/>- `clientAuth`<br/>- `serverAuth`<br/>- `codeSigning`<br/>- `emailProtection`<br/>- `timeStamping`<br/>- `ocspSigning`<br/>- raw OID, for example: \"OID:1.3.6.1.5.5.7.3.2\" means `id-pk 2` which is equivalent to `clientAuth`<br/>Comma-separated string is also supported for validating more than one key usages.<br/>For example, `\"serverAuth,OID:1.3.6.1.5.5.7.3.2\"`",
+      "label": "Verify Key Usage in Cert"
```
Comment (wivwiv) [inline src/schemaText/schema-text-en.json:1391]:
```suggestion
      "label": "Verify Extended Key Usage"
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

**Review by ysfscream — APPROVED** [review]

**Pull request merged**